samples/tab-stage-view/csharp/README.md

```diff
@@ -150,6 +158,18 @@ Please find below demo manifest which is deployed on Microsoft Azure and you can
 
 ![AppOutlook](TabInStageView/Images/AppOutlook.png)
 
+**After opening Outlook web, click the "New mail" button.**
+
+![Open New Mail](Images/OpenNewMail.png)
+
+**on the tool bar on top,select Apps icon. Your sideloaded app title appears among your installed apps**
+
+![OpenAppIcon](Images/OpenAppIcon.png)
+
+**Opening stage view from unfurling link. If you copy and paste a link from https://www.BotFramework.com into the compose message area the link will unfurl.**
```

Comment:
**Opening the stage view from unfurling link. If you copy and paste a link from https://www.BotFramework.com into the compose message area the link will unfurl.**